### PR TITLE
mysql_query: fix failing when single-row query contains commas

### LINKED ISSUE #51: Mysql-Query
##### SUMMARY

Cannot select more than 1 parameters in the Select statement
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

Mysql-query
##### ANSIBLE VERSION

```paste below
ansible 2.10.2
  config file = /home/david/ansible/ansible-repot/ansible.cfg
  configured module search path = ['/home/david/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.5 (default, Jul 28 2020, 12:59:40) [GCC 9.3.0]
```

##### CONFIGURATION

```paste below
None
```

##### OS / ENVIRONMENT

mysql  Ver 15.1 Distrib 5.5.65-MariaDB, for Linux (x86_64) using readline 5.1

##### STEPS TO REPRODUCE

just launch this task 

```yaml
- name: Simple select query to target_table db
  community.mysql.mysql_query:
    login_db: "datab"
    login_password: "0123456789"
    login_user: "9876543210"
    login_host: "localhost"
    query: "Select name, description from target_table "
  register: r_query
```

##### EXPECTED RESULTS

    "query_result": [
        [
            {
                "name": 2,
                "description ": "/srv/abc/ijk/def",
            },
            {
                "name": 2,
                "description ": "/srv/abc/ijk/def",
            },
            {
                "name": 2,
                "description ": "/srv/abc/ijk/def",
            },
            ...
        ]
    ]

##### ACTUAL RESULTS

the module separate in n independant queries

```paste below
...
            "query": [
                "select name",
                " description from target_table "
            ],
            "single_transaction": false
        }
    },
    "msg": "Cannot execute SQL 'select id' args [None]: (1054, \"Unknown column 'name' in 'field list'\")"
```

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fixes https://github.com/ansible-collections/community.mysql/issues/51

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
`plugins/modules/mysql_query.py`

### CODE REVIEW

**Comment by Andersson007** [comment]
@Davidffry @Jorge-Rodriguez 
ready for review

**Comment by Andersson007** [comment]
Now it's green, so ready for review.

@Davidffry @Jorge-Rodriguez , let me explain what i'm doing here.
Before these changes, the query option's type was `list`, so if a value contains commas, Ansible thinks that it is a list.
In this PR, I changed the type to `raw`.
Now if we pass a single-line query, Ansible thinks it is a string and resp. if we pass the value as a list explicitly using the "hyphen" notation, it handles it as a list respectively.
In short, **now we can pass**
```
query: "SELECT first, second, third FROM test_table1"
```
(see the CI tests in Files changed tab) as well as
```
query:
- "SELECT first, second, third FROM test_table1"
- "SELECT first, second, third FROM test_table2"
```

If it's approved, i'll merge and release it ASAP

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!
